torch/onnx/__init__.py

```diff
@@ -165,10 +166,22 @@ def export(
     custom_opsets: Mapping[str, int] | None = None,
     export_modules_as_functions: bool | Collection[type[torch.nn.Module]] = False,
     autograd_inlining: bool = True,
-    **_: Any,  # ignored options
 ) -> ONNXProgram | None:
     r"""Exports a model into ONNX format.
 
+    .. versionchanged:: 2.6
```

Comment:
I don't know how it is rendered but should we merge all of this into two sections?

Reply:
It's rendered like this 
![image](https://github.com/user-attachments/assets/3a2aa1f4-70d1-4173-ac82-30155d18183e) https://docs-preview.pytorch.org/pytorch/pytorch/146923/onnx_torchscript.html#torch.onnx.export

I think it does make sense to combine